src/Uno.UI/UI/Xaml/Controls/StackPanel/StackPanel.Layout.cs

```diff
@@ -51,7 +51,10 @@ protected override Size MeasureOverride(Size availableSize)
 
 		for (int i = 0; i < count; i++)
 		{
-			var view = Children[i];
+			if (Children[i] is not { } view)
```

Comment:
I see situations where one of the children is `null`, causing the app to behave wrongly. More specifically when tapping inside an iOS TextBox.